controller/engine_controller.go

```diff
@@ -607,8 +610,8 @@ func (ec *EngineController) stopMonitoring(e *longhorn.Engine) {
 }
 
 func (m *EngineMonitor) Run() {
-	logrus.Debugf("Start monitoring %v", m.Name)
-	defer logrus.Debugf("Stop monitoring %v", m.Name)
+	m.logger.Debug("Start monitoring engine")
```

Comment:
`Start monitoring`/`Stop monitoring` would be enough.

Reply:
It's better to have `Start monitoring engine` where engine is defined as a field.
This way a search in the logs will be specific.

If you use a generic `start monitoring` in all the different components you will most likely end up with other components.
(ie instance-manager, backupstore, etc)